README.md

```diff
@@ -17,11 +17,10 @@ Build Status
 Tooling & API
 -------------
 
-[Intel TXT Test Suite](cmd/txt-suite) - Test Suite for Intel Trusted Execution Technology validation.
-
-[Intel TXT Provisioning](cmd/txt-prov) - Provisioning of Trusted Platform Module for Intel Trusted Execution Technology usage.
-
-[Intel CBnT Provisioning](cmd/bg-prov) - Provisioning of Converged BootGuard and Trustes Execution Technology (CBnT) usage.
+* [Intel TXT Test Suite](cmd/txt-suite) - Test Suite for Intel Trusted Execution Technology validation.
+* [Intel TXT Provisioning](cmd/txt-prov) - Provisioning of Trusted Platform Module for Intel Trusted Execution Technology usage.
+* [Intel CBnT Provisioning](cmd/bg-prov) - Provisioning of Converged BootGuard and Trusted Execution Technology (CBnT) usage.
+* [pcr0tool](cmd/pcr0tool) - [PCR0](https://security.stackexchange.com/questions/127224/what-does-crtm-refer-to) diagnostics tool.

```

Comment:
Intel pcr0tool

Reply:
We were going to add support of AMD Milan in 2021H1. Should we use prefix "Intel" in this case?

Reply:
Yes only for the intel tooling, If you have another tool for AMD. Go for AMD *

Reply:
Ah I got it for PCR0Tool Then Just use Intel/AMD pcr0tool

Reply:
Fixed.

Reply:
Fixed.